spring-integration-ip/src/main/java/org/springframework/integration/ip/tcp/connection/TcpNioConnection.java

```diff
@@ -663,7 +684,6 @@ protected synchronized void doWrite(ByteBuffer buffer) throws IOException {
 				remaining = buffer.remaining();
 			}
 		}
-
```

Comment:
This is wrong: every member (even last method) of the class has to be surrounded with blank lines.

Reply:
not sure where this empty line is missing

Reply:
Just bring it back and push!
It is not a problem though: we can clean it up on merge.
Just letting you to know what is our code style: https://github.com/spring-projects/spring-integration/wiki/Spring-Integration-Framework-Code-Style !